config/locales/en.yml

```diff
@@ -1,9 +1,9 @@
 en:
   dvl_core:
     footer:
-      about: " is an application by <a href='https://www.dobt.co' target='_blank'>The Department of Better Technology</a>."
-      about_no_app: "Built by <a href='https://www.dobt.co' target='_blank'>The Department of Better Technology</a>."
-      status: Service Status
+      about: " is made by <a href='http://www.dobt.co' target='_blank'>The Department of Better Technology</a>."
```

Comment:
Looks like you didn't properly merge my changes -- these links should be `https`